public/locales/de/translations.json

```diff
@@ -0,0 +1,49 @@
+{
+	"root.description": "Eine Alternative zu dem Nix Ökosystem",
+
+	"header.community": "Gemeinschaft",
+
+	"goals.title": "Die Ziele",
+	"goals.independent.title": "Unabhängigkeit",
+	"goals.independent": "Wir werden Nix, NixPkgs und NixOS forken und betreuen. Zusätzliche Unterstützung für Projekte wie Nix Darwin und Home Manager wird in Betracht gezogen.",
+	"goals.gov.title": "Gesellschaftliche Organisation",
+	"goals.gov": "Zur Verwaltung des Projekts wird ein demokratisches Modell mit gewählten Positionen eingesetzt. Ein Lenkungsausschuss (steering committee) entscheidet über den Kurs, während weitere Ausschüsse spezifische oder logistische Aufgaben übernehmen. Features oder einzelne Projekte werden von Special Interest Groups umgesetzt. SIGs werden in Arbeitsgruppen (Working Groups) zusammenarbeiten, um ihre Ziele übergreifend zu erreichen. Für jedes Komitee und jede Gruppe sind die Ziele, Anforderungen und Managementprozesse klar definiert.",
```

Comment:
I would add an '(SIG)' after the first usage of 'Special Interest Group' to clearly indicate the meaning of the abbreviation.

Reply:
Yikes I thought I had. Fixed.